src/clj/game/cards/resources.clj

```diff
@@ -429,6 +429,18 @@
                              (str "add " (:title c) " to their score area and gain " (get-agenda-points state :runner c)
                                   " agenda point" (when (> (get-agenda-points state :runner c) 1) "s"))))}]}
 
+   "Find the Truth"
+   {:events {:post-runner-draw {:msg (msg "reveal that they drew: "
```

Comment:
Should be "reveals," correct? Using "they" as a singular pronoun is fine in modern English, but the runner's name and his action still need subject-verb agreement.